daphne_worker/src/roles/aggregator.rs

```diff
@@ -123,31 +121,6 @@ impl DapReportInitializer for DaphneWorker<'_> {
             .collect::<Result<Vec<_>, DapError>>()
             .map_err(|e| fatal_error!(err = ?e, "failed to initialize a report"))?;
 
-        let replayed_reports_check = futures::stream::iter(reports_processed_request_data)
-            .map(|(durable_name, reports)| async {
-                durable
-                    .post::<_, HashSet<ReportId>>(
-                        BINDING_DAP_REPORTS_PROCESSED,
-                        DURABLE_REPORTS_PROCESSED_INITIALIZED,
-                        durable_name,
-                        reports,
-                    )
-                    .await
-            })
-            .buffer_unordered(usize::MAX)
-            .try_fold(HashSet::new(), |mut acc, replays| async {
-                acc.extend(replays);
-                Ok(acc)
-            })
-            .await
-            .map_err(|e| fatal_error!(err = ?e, "checking for replayed reports"))?;
-
-        for rep in &mut initialized_reports {
-            if replayed_reports_check.contains(&rep.metadata().id) {
-                rep.reject_due_to(TransitionFailure::ReportReplayed);
-            }
-        }
-
```

Comment:
Please double check that the documentation for the `DapAggregator` trait is up-to-date. IIRC there is some language about checking for replays. Note that we also want to modify `MockAggregator` accordingly.